docs/changelog.rst

```diff
@@ -16,6 +16,17 @@ Changelog
 
   For a complete list of commits, check out the `X.Y.Z`_ release on GitHub.
 
+.. _release 2.1.1:
+
+2.1.1 (2025-MM-DD)
+------------------
+
+New features:
```

Comment:
I'm conflicted about this being a patch release, but also my level of caring is honestly pretty minimal given that it's an experimental feature.